### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by B4rabbas** [waydroid/waydroid#1314]
### Describe the bug

All the video are black, thought youtube, web browser or even reading mp4 files.
![Capture d’écran du 2024-03-06 21-23-22](https://github.com/waydroid/waydroid/assets/36156012/d847ae45-e0c5-4ade-84aa-486d370b6be2)

### Waydroid version

1.4.2

### Device

Linux Desktop

### Operating System

Linux Mint 21.3 x86_64 (with weston v9.0.0)

### Kernel version

5.15.0-97-generic

### Desktop Environment

Cinnamon 6.0.4

### GPU

NVIDIA GeForce RTX 2080 Ti Rev.

### Logs

/var/lib/waydroid/waydroid.cfg
/var/lib/waydroid/waydroid.prop
/var/lib/waydroid/waydroid_base.prop
/var/lib/waydroid/waydroid.log
`sudo waydroid shell -- logcat -d | tee logcat.txt`
`sudo dmesg | tee dmesg.txt`

[waydroid.cfg.txt](https://github.com/waydroid/waydroid/files/14515126/waydroid.cfg.txt)
[waydroid.prop.txt](https://github.com/waydroid/waydroid/files/14515128/waydroid.prop.txt)
[waydroid_base.prop.txt](https://github.com/waydroid/waydroid/files/14515129/waydroid_base.prop.txt)
[waydroid.log.txt](https://github.com/waydroid/waydroid/files/14515131/waydroid.log.txt)
[logcat.txt](https://github.com/waydroid/waydroid/files/14515135/logcat.txt)
[dmesg.txt](https://github.com/waydroid/waydroid/files/14515137/dmesg.txt)

**Comment by Shaarigan** [comment]
Have you tried setting Waydroid into software rendering mode? Nvidia is currently unsupported GPU as from the docs

> Get Waydroid to work through a VM
> This also applies to any unsupported GPU's as well (like nVidia)

https://docs.waydro.id/faq/get-waydroid-to-work-through-a-vm

**Comment by davidbitterlich** [comment]
I can confirm this issue, on my device - using an NVIDIA GPU - I'm also affected by this issue when I try to play a video in Amazon Prime Video App.

I also changed to software rendering and restarted waydroid session without having luck

**Comment by B4rabbas** [comment]
> I can confirm this issue, on my device - using an NVIDIA GPU - I'm also affected by this issue when I try to play a video in Amazon Prime Video App.
> 
> I also changed to software rendering and restarted waydroid session without having luck

Did you've tried yet to play a regular video ? Like a random mp4 file ?

**Comment by TeeCker** [comment]
> Did you've tried yet to play a regular video ? Like a random mp4 file ?

Yes, i tried...In the default Android Gallery Player video is black, works only audio. This is also happening in YouTube. But if i tried playing it with VLC Player everything was working fine. My device: Xeon E3-1200 v3/4th Gen Core Processor Integrated Graphics Controller, driver=i915. Please any clue ?

**Comment by Ag1O1** [comment]
i also have this issue using my nvidia graphics card

**Comment by jmali86** [comment]
Same issue with my installation.
